### ISSUE TITLE
Should we remove the ride value penalty for a second ride of the same type? 

### ISSUE THREAD

**Issue by blackhand1001** [OpenRCT2/OpenRCT2#7376]
If you have two coasters of the same type, the second one will get a ride value penalty for being a duplicate type. This affects dueling coasters. It also affects cases where you might have a boomerang coaster and a normal full circuit corkscrew coaster in the same park. Perhaps this should only apply to flat rides with no track customization?

` // Other ride of same type penalty
    sint32 otherRidesOfSameType = 0;
    Ride *ride2;
    sint32 i;
    FOR_ALL_RIDES(i, ride2) {
        if (ride2->type == ride->type && ride2->status == RIDE_STATUS_OPEN)
            otherRidesOfSameType++;
    }
    if (otherRidesOfSameType > 1)
        value -= value / 4;
`

**Comment by duncanspumpkin** [comment]
People already spam looping coasters. It would be cool if this only applies if the track was identical.

**Comment by Broxzier** [comment]
That can be abused by making a minor change to the track. I'm against removing the penelty entirely. Maybe it could take operating mode into account, or only apply the penalty when the stats like length and drops are very similar.

**Comment by blackhand1001** [comment]
I don't think it does a good enough job deterring people from spamming looping coasters. So the negatives for legit reuse of a coaster type aren't really worth it. I think removing it entirely is probably the best option. It would make having more than one wooden coaster or steel twister in a park which is extremely common in real life not penalized.

**Comment by Gymnasiast** [comment]
I have a better idea: raise the limit from 1 to 2 or 3.

**Comment by jensj12** [comment]
Ideally, building an additional ride should never be penalized, but it should contribute less to the total ride value for every other ride of the same type.
This will work: for every other ride of the same type with a higher ride value, multiply the ride value by a constant (like 0.8). This way the best ride will always have its full ride value, while spamming many rides of the same type will never accumulate more than (in this case) 5 times the ride value of the highest valued ride. The only problem is that (nearly) identical rides will have one penalized more than the other.

**Comment by blackhand1001** [comment]
I think raising the limit to 3 is a good easy compromise.

**Comment by deurklink** [comment]
I think the best solution is to count dueling rides as 0.5 rides. In reality guests also prefer a big variation of rides so I don't think raising the value to 3 is a good solution. 

For wooden coasters this could be debated though as the only variety in rct2 comes from the types of car used.

**Comment by blackhand1001** [comment]
What about steel twister coasters though. They all use the same ride type. Obviously a floorless and a sitdown hyper aren't the same type of ride. Great adventure has 3 steel twister coasters in real life.

**Comment by deurklink** [comment]
Maybe we should let it check car type then instead of ride type.
If you then count rides which are synchronized with another station as half a ride, I think you have all cases covered.

**Comment by Krutonium** [comment]
You can have up to (iirc) 16 rides synchronized - perhaps it would be better to just count any synchronized rides as a single ride instead?

**Comment by deurklink** [comment]
> You can have up to (iirc) 16 rides synchronized - perhaps it would be better to just count any synchronized rides as a single ride instead?

That would probably be the easiest to implement.
